actors.md

```diff
@@ -1091,6 +1092,46 @@ func ChangeWorker(addr Address) {
 }
 ```
 
+#### `PaymentVerify`
+
+Verifies a storage market payment channel voucher's 'Extra' data.
+
+**Parameters**
+
+```sh
+type PaymentVerify struct {
+    Extra StorageVoucherData
+    Proof Bytes
+} representation tuple
+
+type StorageVoucherData struct {
+    DataCommitment Bytes
+    RequiredSector Bytes
+} representation tuple
+```
+
+**Algorithm**
+
+```go
+func PaymentVerify(extra StorageVoucherData, proof []byte) {
```

Comment:
This method has utility for local determination of piece inclusion as well as payment verification on-chain. Consider renaming `PaymentVerify` to `VerifyPieceInclusion` and `StorageVoucherData` to `PieceCommitmentData`. Also `proof` probably needs to be a struct that contains a PIP and a sector ID.

Reply:
We can't make the types unique to the storage miner, otherwise its not actually pluggable. We could deserialize the proof bytes into a struct though.

+1 on the rename